GlobalSuppressions.cs

```diff
@@ -0,0 +1,6 @@
+// This file is used by Code Analysis to maintain SuppressMessage
+// attributes that are applied to this project.
+// Project-level suppressions either have no target or are given
+// a specific target and scoped to a namespace, type, member, etc.
+
+[assembly: System.Diagnostics.CodeAnalysis.SuppressMessage("Style", "IDE0022:Use expression body for methods", Justification = "It's for better understanding and for simpler debugging")]
```

Comment:
Why is this necessary?

Reply:
because now we don't get warnings for methods like this:

```csharp
public void blup(){
   Console.WriteLine("hello world");
}
```

Because this can be converted to:

```csharp
public void blup => Console.WriteLine("hello world");
```

But I hat this because you can't debug this single line good. And it is harder to read. explicit in long method calls.

Reply:
I will update the EditorConfig later.
that should solve this problem